### ISSUE TITLE
Can this work as a submit button in a form?

### ISSUE THREAD

**Issue by danoc** [danoc/clickable-box#4]

**Comment by gnapse** [comment]
Perhaps it should be mentioned in the README as one of the conditions under which this should not be used?

**Comment by danoc** [comment]
Good idea! Added it here: https://github.com/danoc/clickable-box/commit/09153a21b0985a2746580b29f3d48cd931be37f8
